### PR TITLE
Got most of controller and other reservation functions working

### PR DESCRIPTION
**Pull request by matthewgrossman**

### CODE REVIEW

**Comment by matthewgrossman** [comment]
leaving comments in the mornig

File: controllers/reservations.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ var express = require('express'),
```
Comment (kenzshelley) [inline controllers/reservations.js]: OMG YOU USED 4 SPACE INDENTS that's against the style guidelines geeze matt

File: tests/models/reservation.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+var Firebase = require('Firebase'),
```
Comment (kenzshelley) [inline tests/models/reservation.js]: UGH 4 spaces again why can't you learn @matthewgrossman

File: tests/refresh_db.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ var addSpots = function() {
             "Kenz's House")
```
Comment (kenzshelley) [inline tests/refresh_db.js]: AND AGAIN

**Comment by kenzshelley** [comment]
Just left some *really* helpful comments. L :goat: TM

**Comment by kenzshelley** [comment]
:fu: it :ship: it

**Pull request merged**